torch/quantization/default_mappings.py

```diff
@@ -15,6 +15,7 @@
     nn.Linear: nnq.Linear,
     nn.ReLU: nnq.ReLU,
     nn.ReLU6: nnq.ReLU6,
+    nn.Conv1d: nnq.Conv1d,
```

Comment:
Do we need to change for qat mapping as well?

Reply:
We don't have qat conv1d

Reply:
I will send qat conv1d later (P1 for now :))